code/modules/clothing/suits/f13factionarmor.dm

```diff
@@ -330,7 +330,7 @@
 	w_class = WEIGHT_CLASS_SMALL
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
 
-/*
```

Comment:
Don't just comment code out.
If you remove from the game then remove from the code and remove the icons as well.
That way you avoid bloat.

Reply:
Someone asked to put this stuff back, because someone removed sometime ago